tools/make_patch.c

```diff
@@ -177,7 +189,7 @@ struct symbol *parse_symfile(FILE *file, const char *prefix)
 		case ';':
 		case '\n':
 			// If we encounter a newline or comment,
-			//   we've reached the end of the label name.
+			// we've reached the end of the label name.
```

Comment:
Indenting continuations of the same sentence is just my personal style, but if you've removed it across the entire file that's ok.

Reply:
https://github.com/pret/pokecrystal/pull/882/commits/3a78bcc8073928f2675d66b38124b90b6796ffca Made comment indenting consistent.